monitoring/harmony-monitor/blockchain-watchdog/rpc.go

```diff
@@ -4,12 +4,13 @@ package main
 const (
 	NodeMetadataRPC   = "hmy_getNodeMetadata"
 	BlockHeaderRPC    = "hmy_latestHeader"
-	PendingCxRPC      = "hmy_getPendingCxReceipts"
+	PendingCXRPC      = "hmy_getPendingCXReceipts"
 	SuperCommitteeRPC = "hmy_getSuperCommittees"
+	LastCrossLinkRPC  = "hmy_getLastCrossLinks"
 	JSONVersion       = "2.0"
 )
 
-// NodeMetadataReply
+// Node Metadata RPC Reply
```

Comment:
install golint as a plugin for editor, this is actually golang convention you're going against.